dev-packages/e2e-tests/test-applications/default-browser/tests/transactions.test.ts

```diff
@@ -0,0 +1,62 @@
+import { expect, test } from '@playwright/test';
+import { waitForError, waitForTransaction } from '@sentry-internal/test-utils';
+
+test('sends a pageload transaction', async ({ page }) => {
```

Comment:
this test is redundant I think, given we have the test in `performance.test.ts`

Reply:
gotcha